source/docs/third-party-services/comments.md

```diff
@@ -116,6 +116,48 @@ Valine is a fast, simple & efficient Leancloud based no back end comment system.
       serverURLs: # When the custom domain name is enabled, fill it in here (it will be detected automatically by default, no need to fill in)
     ```
 
+### MiniValine (China Modified from Valine)
```

Comment:
hi, can you just use MiniValine as title to make hyperlinks working. `/docs/third-party-services/comments#MiniValine`
as well as Valine (China), Changyan (China) to Valine, Changyan

Reply:
oh, no, I mean changing the titles in `/docs/third-party-services/comments.md`.
the title must be same as anchor.

Reply:
well, i saw the second commit. em, that would work too.

Reply:
https://5e81b47f200691000a5bd29d--theme-next.netlify.com/docs/third-party-services/

It has been confirmed that anchor links seem to be working properly.

Reply:
but it seems better for explation (china) in index.md. well, forget it.